airbyte-integrations/bases/base-java/src/test/java/io/airbyte/integrations/base/ssh/SshTunnelTest.java

```diff
@@ -24,7 +24,6 @@
 import org.junit.jupiter.params.provider.ValueSource;
 
 class SshTunnelTest {
-
```

Comment:
does this pass `SUB_BUILD=CONNECTORS_BASE ./gradlew format`?

Reply:
it does